utilities/blob_db/blob_db_impl.cc

```diff
@@ -2244,6 +2221,39 @@ Iterator* BlobDBImpl::NewIterator(const ReadOptions& opts,
                             column_family, this);
 }
 
+Status DestroyBlobDB(const std::string& dbname, const Options& options,
+                     const BlobDBOptions& bdb_options) {
+  const ImmutableDBOptions soptions(SanitizeOptions(dbname, options));
+  Env* env = soptions.env;
+
+  Status status;
+  std::string blobdir;
+  blobdir = (bdb_options.path_relative) ? dbname + "/" + bdb_options.blob_dir
+                                        : bdb_options.blob_dir;
+
+  std::vector<std::string> filenames;
+  env->GetChildren(blobdir, &filenames);
+
+  for (const auto& f : filenames) {
+    uint64_t number;
+    FileType type;
+    if (ParseFileName(f, &number, &type) && type == kBlobFile) {
+      Status del = env->DeleteFile(blobdir + "/" + f);
+      if (status.ok() && !del.ok()) {
+        status = del;
+      }
+    }
+  }
+  env->DeleteDir(blobdir);
+
+  Status destroy = DestroyDB(dbname, options);
+  if (status.ok() && !destroy.ok()) {
+    status = destroy;
```

Comment:
Same comment as above. Execution never reaches 2251 if there was `DeleteFile` already failed in line 2241. `destroy` status is just discarded. 
How important is it to return multiple failure statuses from this `DestroyBlobDB` function to the caller?